concord-server/server/src/main/java/com/rabidgremlin/concord/dao/UploadDao.java

```diff
@@ -0,0 +1,45 @@
+package com.rabidgremlin.concord.dao;
+
+import com.rabidgremlin.concord.api.UnlabelledPhrase;
+import org.apache.commons.codec.digest.DigestUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.jdbi.v3.sqlobject.CreateSqlObject;
+import org.jdbi.v3.sqlobject.transaction.Transaction;
+
+import java.util.List;
+
+public interface UploadDao {
+
+    @CreateSqlObject
+    VotesDao votesDao();
+
+    @CreateSqlObject
+    PhrasesDao phrasesDao();
+
+    @Transaction
+    default void uploadUnlabelledPhrases(List<UnlabelledPhrase> unlabelledPhrases) {
+
+        for(UnlabelledPhrase unlabelledPhrase : unlabelledPhrases)
+        {
+            // skip header
```

Comment:
Pull this out of here. It is a quirk of the .csv file can keep that detail in the resource.